src/annotations/api/index.ts

```diff
@@ -0,0 +1,27 @@
+// Types
+import {PostAnnotationResponse, PostAnnotationPayload} from 'src/types'
+
+// Libraries
+import axios, {AxiosResponse} from 'axios'
+
+// Constants
+import {API_BASE_PATH} from 'src/shared/constants'
+
+// URL
+const url = `${API_BASE_PATH}api/v2private/annotations`
+
+export const writeAnnotation = async (
+  annotations: PostAnnotationPayload[]
+): Promise<AxiosResponse<PostAnnotationResponse[]>> => {
+  try {
+    const res = await axios.post(url, annotations)
+
+    if (res.status !== 200) {
```

Comment:
What happens if this returns a 201, or a 202 or a 204, which are all potentially valid here?

https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#2xx_success

2xx codes are successes, so we usually do something [like this](https://github.com/influxdata/ui/blob/master/src/templates/api/index.ts#L539)